Manifest.toml

```diff
@@ -1,8 +1,8 @@
 # This file is machine-generated - editing it directly is not advised
 
-julia_version = "1.8.5"
+julia_version = "1.8.2"
 manifest_format = "2.0"
-project_hash = "5599a2430900dfa80e9ac9553b2cce7160e22d89"
+project_hash = "ef0cd293c932a3ed19df417102b084654511cd0b"
```

Comment:
You probably don't want to commit the `Manifest.toml`. Most project don't do that because it constrains the dependencies unnecessarily much. Just make sure that you have `[compat]` set for all dependencies in `Project.toml`.

Reply:
Thanks! I've removed it.